mm2src/coins/solana/solana_tests.rs

```diff
@@ -335,3 +337,112 @@ fn solana_test_tx_history() {
     }
     log!("{}", serde_json::to_string(&history).unwrap());
 }
+
+#[test]
+fn solana_coin_send_and_refund_maker_payment() {
+    let passphrase = "federal stay trigger hour exist success game vapor become comfort action phone bright ill target wild nasty crumble dune close rare fabric hen iron".to_string();
+    let (_, coin) = solana_coin_for_test(passphrase, SolanaNet::Devnet);
+    let solana_program_id = "AEwou7VU9bahWRSN2zzgMGqozNY2G27U9fbbdLF9myZx";
+    let solana_program_id = bs58::decode(solana_program_id).into_vec().unwrap_or_else(|e| {
+        eprintln!("Failed to decode program ID: {}", e);
+        Vec::new()
+    });
```

Comment:
After merging the satomic swap PR we need to deploy the latest program and use it here.

Reply:
@shamardy The latest Solana program ID, deployed after the recent changes in the Solana repository, is 3fystoi7pB1cnDEbRRpSjFJA4fG3W2vQQZ21jSrBc11B. I’ve already updated it in the code.